tools/tests/test_cases/test_gemm.cpp

```diff
@@ -171,3 +172,75 @@ TEST_F(DnnlPluginNext_GEMM, LLaMa_v2_case_2)
     params_.allow_fp16_computations = true;
     run();
 }
+
+TEST_F(DnnlPluginNext_GEMM, TransposedCases_Normal_FP32)
+{
+    params_.shape_a = TensorShape(1, 1, 32, 128);
```

Comment:
In DML dispatcher we swap dimensions, so tensor shapes in tests will not work with dml dispatcher?